java/java.editor/test/unit/src/org/netbeans/modules/editor/java/GoToSupportTest.java

```diff
@@ -1282,4 +1297,223 @@ private static boolean hasPatterns() {
             return false;
         }
     }
+    
+    public void testRecords1() throws Exception {
+//        try {
```

Comment:
This should be (presumably?) uncomentted. (Dtto the other occurrences below.)

Reply:
Comment addressed.